website/common/script/content/constants/events.js

```diff
@@ -11,10 +11,18 @@ const gemsPromo = {
 export const EVENTS = {
   noEvent: {
     start: '2023-08-01T23:59-04:00',
-    end: '2023-08-31T08:00-04:00',
+    end: '2023-08-10T08:00-04:00',
     season: 'normal',
     npcImageSuffix: '',
   },
+  bundle202308: {
+    start: '2023-08-15T23:59-04:00',
```

Comment:
We usually start these things at 8:00, not midnight, yes?

Reply:
Just updated times for potions and pet quest bundle